<launch>
  <node pkg="visual_servoing" exec="pose_visualization.py" name="pose_visualization" output="screen">
	  <!--subscriber topic parameter setting/-->
    <param name="odom_topic" value="/wheel_odom" /><!--里程計 Topic/-->
    <param name="apriltag_topic" value="/apriltag_poses" /><!--AprilTag的 Topic/-->
    <param name="pallet_topic" value="/pallet" /><!--棧板或目標的 Topic/-->
    <param name="forkpose_topic" value="/forklift_pose" /><!--牙叉編碼器 Topic/-->

  </node>
</launch>
